src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_TupleCreationExpression.cs

```diff
@@ -46,25 +51,25 @@ private BoundNode RewriteTupleCreationExpression(BoundTupleExpression node, Immu
                 var smallestCtor = (MethodSymbol)TupleTypeSymbol.GetWellKnownMemberInType(smallestType.OriginalDefinition,
                                                                                             TupleTypeSymbol.GetTupleCtor(smallestType.Arity),
                                                                                             _diagnostics,
-                                                                                            node.Syntax);
+                                                                                            syntax);
                 if ((object)smallestCtor == null)
                 {
-                    return node;
+                    return null;
```

Comment:
Original code returned original node in case of a failure. Bad node is probably just as good. Perhaps better, so that we do not try lowering any further.